src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,39 @@
+package domain;
+
+import java.util.List;
+
+public class Lotto {
+
+  private final List<Integer> numbers;
+
+  public Lotto(List<Integer> numbers) {
+    this.numbers = numbers;
```

Comment:
잘 보니 전반적으로 들여쓰기가 +2개의 빈 칸으로 되어있으시군요? 
[자바 코딩 컨벤션](https://myeonguni.tistory.com/1596)에 따르면 `+4개의 빈칸`으로 **들여쓰기**를 진행해야하는 것으로 알고있어요. 
IntelliJ의 설정을 한번 살펴봐주시면 좋을 것 같습니다. 

```suggestion
      this.numbers = numbers;
```